.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: java
+
+install:
+  - . $HOME/.nvm/nvm.sh
+  - nvm install 8.9.0
+  - nvm use 8.9.0
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
why are we skipping tests?

Reply:
Tests are executed in a later phase using `mvn test`.